beanstalk_dispatch/client.py

```diff
@@ -1,12 +1,24 @@
+import base64
 import json
 
-import boto.sqs
+import boto3
+import six
+from botocore.exceptions import ClientError
 from django.conf import settings
 
 from .common import create_request_body
 from .execution import execute_function
 
 
+def encode_message(value):
+    """
+    Encode the message in the same way boto2 did with queue.write(message)
+    """
+    if not isinstance(value, six.binary_type):
```

Comment:
I suspect we already rely on Python 3 via other dependencies, so you can avoid `six` and do something like `isinstance(value, bytes)` here.

Reply:
Done